### DISCUSSION TITLE
@nrwl/react:setup-ssr does not support Vite apps

### DISCUSSION THREAD

**Discussion by a777med** [nrwl/nx#22693] [Feature Requests]
- [x] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Description

Runnig `nx generate setup-ssr` on a basic React Vite app works, but when trying to run `nx serve-server cms`, there's an error. 
```
Cannot find module 'C:\Users\user\portal\apps\cms\webpack.config.js'
```                 

## Motivation

Apart from the above error, I think using Vite's SSR capability or an SSR plugin for Vite would make more sense.

## Suggested Implementation

Vite SSR

## Alternate Implementations

Make it work with Webpack

**Comment by mandarini** [comment]
Hi there @a777med ! Thanks for creating a feature request. I see you are willing to implement this. Please go ahead and create a PR, and tag me so that I can review! :)

**Comment by condini-mastheus** [comment]
@a777med how can I help with this issue? I would like to join and try to implement this feature also

**Comment by a777med** [comment]
Hi @condini-mastheus , 
Unfortunately, I haven't had the time to implement it yet. You can start on your own.
Maybe @mandarini can guide us in the right direction.

**Comment by jigarjoshi011** [comment]
i using nx and i need to have react vite having ssr feature into it but unable to create this kind of app
